funnel/views/commentvote.py

```diff
@@ -1,13 +1,17 @@
 # -*- coding: utf-8 -*-
 
-from flask import redirect, g, flash, abort, jsonify, request
-from coaster.views import jsonp, route, requires_permission, UrlForView, ModelView
+from datetime import datetime
+from flask import redirect, flash, abort, jsonify, request, render_template
+from coaster.auth import current_auth
+from coaster.views import jsonp, route, requires_permission, UrlForView, ModelView, requestargs
 from baseframe import _, forms
 
 from .. import app, funnelapp, lastuser
-from ..models import db
-from .mixins import ProposalViewMixin, CommentViewMixin
+from ..forms import CommentForm, DeleteCommentForm
+from ..models import db, Comment
 from .decorators import legacy_redirect
+from .helpers import send_mail
+from .mixins import ProposalViewMixin, CommentViewMixin
```

Comment:
If mixins are not being reused across models, they should be in the respective file for each model. `CommentViewMixin` should be in `views/comments.py`, etc.